lib/hanami/config/logger.rb

```diff
@@ -89,8 +120,16 @@ def app_name(value = nil)
       # @see Hanami::Config::Logger#stream
       # @see Hanami::Config::Logger#engine
       def build
-        @engine ||
-          ::Hanami::Logger.new(@app_name, device: @device)
+        logger = ::Hanami::Logger.new(@app_name, device: @device)
+        logger.level = @level
+
+        @engine || logger
+      end
+
+    private
+
+      def detect_level(level)
+        level.is_a?(Integer) ? level : LEVEL_HASH[level.to_sym.downcase]
```

Comment:
@davydovanton What if we let `Hanami::Logger` to check this? If you think, we're using too much knowledge of its implementation. We analyize integer and symbols (see `LEVEL_HASH`) on its behalf. This change would make `Hanami::Logger` capable to take the right decision by itself. :wink: